server/index.js

```diff
@@ -3,39 +3,34 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const path = require('path');
 const AWS = require('aws-sdk');
+const cron = require('node-cron');
+const winston = require('winston');
+const calc = require('./calc');
 const db = require('../database/database');
 const elastic = require('../dashboard/elastic');
-const calc = require('./calc');
-const cron = require('node-cron');
+const sessionData = require('../data-simulation/sessionData');
 
-AWS.config.loadFromPath(path.resolve('credentials/config.json'));
+const url = {
+  sessionsQueue: 'https://sqs.us-west-2.amazonaws.com/287554401385/tetraflix-sessions.fifo',
+  usersQueue: 'https://sqs.us-west-2.amazonaws.com/287554401385/tetraflix-userprofiles.fifo',
+};
 
-const sessionsQueueUrl = 'https://sqs.us-west-1.amazonaws.com/287554401385/tetraflix-sessions-fifo';
-const usersQueueUrl = 'https://sqs.us-west-1.amazonaws.com/287554401385/tetraflix-userprofiles-fifo';
-
-const movieGenres = [
-  'action',
-  'animation',
-  'comedy',
-  'documentary',
-  'drama',
-  'family',
-  'fantasy',
-  'horror',
-  'international',
-  'musical',
-  'mystery',
-  'romance',
-  'sci_fi',
-  'thriller',
-  'western',
-];
+AWS.config.loadFromPath(path.resolve('credentials/config.json'));
 
 const sqs = new AWS.SQS();
 sqs.sendMessageAsync = Promise.promisify(sqs.sendMessage);
 sqs.receiveMessageAsync = Promise.promisify(sqs.receiveMessage);
 sqs.deleteMessageAsync = Promise.promisify(sqs.deleteMessage);
 
+const logger = winston.createLogger({
```

Comment:
Are you rotating your log files, or do you just keep appending to the same file?

Reply:
Currently I am not rotating my log files.  If I decide to add that feature, I am planning to use winston-daily-rotate-file module to rotate it for the time intervals I need.  Thanks for noting!